apps/financial-aid/web-osk/src/components/Status/InProgress.tsx

```diff
@@ -41,7 +41,7 @@ const InProgress = ({ currentApplication }: Props) => {
             cta={{
               label: 'Hlaða upp gögnum',
               onClick: () => {
-                router.push(`${router.query.id}/gogn`)
+                router.push(`stada/${router.query.id}/gogn`)
```

Comment:
Should we maybe have all routes in a const file? Then in the future if we add something like `stada` then we only change that file.